apollo/agent/serde.py

```diff
@@ -48,3 +55,27 @@ def default(self, obj: Any):
         if serialized is not obj:  # serialization happened
             return serialized
         return super().default(obj)
+
+
+def row_value_encoder(value: Any) -> Any:
```

Comment:
Encoding is different here than the default agent serializer, we want it to be more lightweight, and typing to be inferred if needed from field schema (which we still need to add...).